### DISCUSSION TITLE
Getting disconnected

### DISCUSSION THREAD

**Discussion by Edmonas** [Chia-Network/chia-blockchain#5214] [Support]
I can't understand what's wrong with syncing. It gets synced, for few seconds or a minute, and again disconnects :(
![Disconnected](https://user-images.githubusercontent.com/84113476/118402610-4a5c7680-b673-11eb-8127-d003efc57001.PNG) Port 8444 is opened, and i get this error in the debug log 
2021-05-16T18:13:42.156 full_node full_node_server        : ERROR    Exception:  <class 'concurrent.futures._base.CancelledError'>, closing connection {'host': '127.0.0.1', 'port': 8449}. Traceback (most recent call last):
  File "chia\server\server.py", line 531, in api_call
  File "asyncio\tasks.py", line 435, in wait_for
concurrent.futures._base.CancelledError
![8444](https://user-images.githubusercontent.com/84113476/118402783-277e9200-b674-11eb-8ce4-c33525b986c1.PNG)
![Debug1](https://user-images.githubusercontent.com/84113476/118402790-2c434600-b674-11eb-9b7d-1b6b6d9b83ac.PNG)
Maybe you have any suggestions how to solve this issue?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
